src/@next/Bar/Bar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Colors } from '..';
+import { ComponentAction } from '../../types/componentAction';
+import { Button, PrimaryButton } from '../Button';
+import { ButtonGroup } from '../ButtonGroup';
+import { Typography } from '../Typography';
+import {
+  StyledBarActionWrapper,
+  StyledBar,
+  StyledBarHeaderWrapper,
+  StyledBarContainer,
+} from './BarStyle';
+
+export type BarProps = {
+  heading: React.ReactNode;
+  subheading?: React.ReactNode;
+  primaryAction: ComponentAction;
+  secondaryAction?: ComponentAction;
+  tertiaryAction?: ComponentAction;
+  position?: 'top' | 'bottom';
+};
+
+export const Bar = React.forwardRef<HTMLDivElement, BarProps>(function Bar(
+  {
+    heading,
+    subheading,
+    primaryAction,
+    secondaryAction,
+    tertiaryAction,
+    position,
+  }: BarProps,
+  ref
+) {
+  const headerMarkup = () => {
+    if (subheading) {
+      return (
+        <>
+          <Typography as="div" variant="body2">
+            {heading}
+          </Typography>
+          <Typography as="div" variant="subtitle2" color={Colors.Neutral.B40}>
+            {subheading}
+          </Typography>
+        </>
+      );
+    }
+    return (
+      <Typography as="div" variant="headline6">
+        {heading}
+      </Typography>
+    );
+  };
+
+  return (
+    <StyledBar data-align={position} ref={ref}>
+      <StyledBarContainer>
+        <StyledBarHeaderWrapper>{headerMarkup()} </StyledBarHeaderWrapper>
+        <StyledBarActionWrapper>
+          <ButtonGroup>
```

Comment:
button size should be medium for mobile view (max-width: 768px)

Reply:
@ninariccimarie Is this automatically handled, just like Typography?